spec/features/merchants/dashboard_spec.rb

```diff
@@ -21,75 +21,104 @@
     expect(current_path).to eq("/merchants/#{merchant.id}/items")
   end
 
-  it "has the names of the top 5 customers with largest number of completed transactions" do
-    merchant = create(:merchant)
-
-    customer_1 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 6, first_name: 'Bob')
-    customer_2 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 3, first_name: 'John')
-    customer_3 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 8, first_name: 'Abe')
-    customer_4 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 1, first_name: 'Zach')
-    customer_5 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 4, first_name: 'Charlie')
-
-    visit "/merchants/#{merchant.id}/dashboard"
-
-    within 'div.top_customers' do
-      expect('Abe').to appear_before('Bob')
-      expect('Bob').to appear_before('Charlie')
-      expect('Charlie').to appear_before('John')
-      expect('John').to appear_before('Zach')
+  describe 'top 5 customers section' do
+    it "has the names of the top 5 customers with largest number of completed transactions" do
+      merchant = create(:merchant)
+
+      customer_1 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 6, first_name: 'Bob')
+      customer_2 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 3, first_name: 'John')
+      customer_3 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 8, first_name: 'Abe')
+      customer_4 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 1, first_name: 'Zach')
+      customer_5 = create(:customer_with_transactions, merchant: merchant, transaction_result: 0, transaction_count: 4, first_name: 'Charlie')
+
```

Comment:
The power of facotry bot on full display! Thank you for taking the time over break to learn that, and for bringing that knowledge to us!! So badass to see it flex it's muscle in this way!